GOVERNANCE.md

```diff
@@ -0,0 +1,136 @@
+# Debezium Governance
+
+This document defines the project governance for the [Debezium](https://github.com/debezium) project.
+
+## Overview
+
+Anyone can contribute to Debezium by following the [contribution workflow](https://github.com/debezium/debezium/CONTRIBUTE.md) or discussing features and improvements on GitHub issues.
+
+## Project Roles
```

Comment:
Should this also define a "Contributor" role? I.e. a community member e.g. proposing a PR, but who are not a committer (yet).

Reply:
I'll move that definition/description up from "Debezium Conmitters".

Reply:
Let's define "Contributor" role that active people automatically get.  It's better to be explicit than implicit.